services/ui-src/src/components/layout/Footer.tsx

```diff
@@ -4,9 +4,9 @@ import { Box, Container, Flex, Image, Link, Text } from "@chakra-ui/react";
 // utils
 import { useBreakpoint } from "utils";
 // assets
-import hhsLogo from "assets/logos/logo_hhs.svg";
+import hhsLogo from "assets/logos/logo_hhs.png";
 import mcrLogo from "assets/logos/logo_mcr_footer.png";
-import medicaidLogo from "assets/logos/logo_medicaid.svg";
+import medicaidLogo from "assets/logos/logo_medicaid.png";
```

Comment:
png vs svg could be related to the squishing

Reply:
On this one I would probably stick with svg 'cause it'll give us higher quality.

Reply:
done! 💯